### PR TITLE
luci-app-adblock-fast: sync with principal

### PR DESCRIPTION
**Pull request by stangri**
Maintainer: me
Compile tested: x86_64, Dell EMC Edge620, OpenWrt 23.05.5
Run tested: x86_64, Dell EMC Edge620, OpenWrt 23.05.5

### CODE REVIEW

**Review by systemcrash — APPROVED** [review]

**Comment by stangri** [comment]
Thank you @systemcrash, I forgot about these PRs. ;)

**Comment by hnyman** [comment]
And please remember to also keep 24.10 updated. 
Especially on the packages repo.

 (I have been so far keeping luci 24.10 in sync with master by fast-forwaring, but gradually there starts to be real differences between master and 24.10, so that fast-foward will stop soon. Packages feed repo is already fully in backport to 24.10 mode.)

**Comment by stangri** [comment]
> And please remember to also keep 24.10 updated. Especially on the packages repo.
> 
> (I have been so far keeping luci 24.10 in sync with master by fast-forwaring, but gradually there starts to be real differences between master and 24.10, so that fast-foward will stop soon. Packages feed repo is already fully in backport to 24.10 mode.)

Thank you for the reminder Hannu. Until 24.10 is released, would it be acceptable to create PRs directly against 23.05 and 24.10 branches, same way PRs are created for snapshots, rather than cherry-picking commits from snapshots to releases? That would be much easier for me to do that go thru commits to cherry-pick the missing ones when catching up.

**Comment by hnyman** [comment]
Yeah, PR are naturally ok for 24.10

File: applications/luci-app-adblock-fast/htdocs/luci-static/resources/adblock-fast/status.js [hunk 1]
```diff
@@ -198,7 +198,7 @@ var status = baseclass.extend({
 							"Use of external dnsmasq config file detected, please set '%s' option to '%s'"
 						).format("dns", "dnsmasq.conf"),
 						warningMissingRecommendedPackages: _(
-							"Some recommended packages are missing"
+							"Missing recommended package: '%s'"
```
Comment (systemcrash) [inline applications/luci-app-adblock-fast/htdocs/luci-static/resources/adblock-fast/status.js:201]: I must've missed this - is this missing a `.format()`?

Comment (stangri) [reply applications/luci-app-adblock-fast/htdocs/luci-static/resources/adblock-fast/status.js:201]: It isn't missing the `.format()`, the entire text label is assembled in https://github.com/openwrt/luci/blob/ac1043cdc6a24485a052136b5be1598a21152a52/applications/luci-app-adblock-fast/htdocs/luci-static/resources/adblock-fast/status.js#L318-L319 where the `.format()` is present.

**Pull request merged**